sdk/include/system/DateTime.hpp

```diff
@@ -246,6 +250,9 @@ class TimeDuration final
 class DateTime final
 {
 public:
+
+   static constexpr char const* ISO_8601_INPUT_FORMAT  = "%Y-%m-%dT%H:%M:%S%F%ZP";
```

Comment:
If we move the impl of `format` to the CPP, we should be able to move this as well.

Reply:
This was not recognized when placed in the source file.